cms/envs/common.py

```diff
@@ -904,6 +904,7 @@
 
 SOUTH_MIGRATION_MODULES.update({
     'edx_notifications': 'edx_notifications.stores.sql.migrations',
+    'problem_builder': 'problem_builder.south_migrations',
 })
```

Comment:
@antoviaque I thought workbench is running Django 1.7. If it's not - than my guess on why @bradenmacdonald used `south_migrations` is wrong, but I believe he had some reason to do that.